packages/ember-application/tests/system/dependency_injection/to_string_test.js

```diff
@@ -32,7 +32,7 @@ QUnit.module('Ember.Application Dependency Injection – toString', {
   }
 });
 
-QUnit.test('factories', function() {
+QUnit.test('factories', () => {
```

Comment:
Tests have their own context, lets not use `() => {}` here. Please change these back to `function() { }`.